etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messagelink/MessageLinkException.java

```diff
@@ -0,0 +1,13 @@
+package gov.hhs.cdc.trustedintermediary.etor.messagelink;
+
+/** This exception is thrown when there is an error linking messages. */
+public class MessageLinkException extends Exception {
+
+    public MessageLinkException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public MessageLinkException(String message) {
+        super(message);
+    }
```

Comment:
This constructor can be removed because it is only called in tests.  Those tests that call this constructor can instead call the other constructor and pass in `new Exception()` for the second argument.